css/layout.css

```diff
@@ -121,9 +124,12 @@ hr {
   margin-bottom: 0;
 }
 
+.page-header {
+  text-align: center;
+}
 /*--------------Nav Layout-------------------*/
 .nav-horizontal li {
-  float: left;
+  float: right;
```

Comment:
Could floats be abstracted into a utility style rather than being so specific to this selector?